sdk/batch/Azure.Compute.Batch/tests/Infrastructure/WindowsPoolFixture.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Azure.Compute.Batch.Tests.Infrastructure
+{
+    internal class WindowsPoolFixture : PoolFixture
+    {
+        public WindowsPoolFixture(BatchClient batchClient, string poolID, bool isPlayback) : base(poolID, batchClient, isPlayback) { }
+
+        public async Task<BatchPool> CreatePoolAsync(int targetDedicatedNodes = 1)
+        {
+            BatchPool currentPool = await FindPoolIfExistsAsync();
+
+            if (currentPool == null)
+            {
+                BatchPoolCreateContent batchPoolCreateOptions = CreatePoolOptions(targetDedicatedNodes);
+                Response response = await client.CreatePoolAsync(batchPoolCreateOptions);
+                if (response == null)
```

Comment:
Was the empty clause intentional?   I don't see what value this provides.

Reply:
no, removed